zookeeper-contrib/zookeeper-contrib-rest/src/test/java/org/apache/zookeeper/server/jersey/RestTestSuite.java

```diff
@@ -18,23 +18,23 @@
 
 package org.apache.zookeeper.server.jersey;
 
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.platform.suite.api.SelectClasses;
 import org.junit.runner.RunWith;
-import org.junit.runners.Suite;
-import org.junit.runners.Suite.SuiteClasses;
 
-@RunWith(Suite.class)
-@SuiteClasses({WadlTest.class, GetTest.class, GetChildrenTest.class,
+@RunWith(JUnitPlatform.class)
+@SelectClasses({WadlTest.class, GetTest.class, GetChildrenTest.class,
     CreateTest.class, SetTest.class, ExistsTest.class, DeleteTest.class })
 public class RestTestSuite {
```

Comment:
This doesn't seem to be used.  The suite runs as well without it, and actually runs nine tests (with or without) as opposed to the seven listed here.